rake/site.rake

```diff
@@ -84,9 +84,12 @@ namespace :site do
     siteify_file(".github/SUPPORT.markdown", "title" => "Support")
   end
 
-  desc "Write the site latest_version.txt file"
-  task :version_file do
-    File.open("#{docs_folder}/latest_version.txt", "wb") { |f| f.puts(version) } unless version =~ %r!(beta|rc|alpha)!i
+  desc "Write the site latest version"
```

Comment:
How about..?
```diff
- "Write the site latest version"
+ Write the latest Jekyll version used to generate the site"
```

Reply:
I kept the beginning of your suggestion, site is generated by GitHub Pages and as now, it may not use latest version.